package.json

```diff
@@ -42,7 +42,8 @@
     "format": "prettier frontend --check",
     "format:write": "yarn format --write",
     "lint": "eslint --ext js,ts,tsx frontend --max-warnings=0",
-    "test:backend": "jest --config=test/jest-config/jest.config-backend.js",
+    "test:backend": "jest --config=test/jest-config/jest.config-backend.js -t --testPathIgnorePatterns='.*\\/generate-transactions\\.spec\\.ts'",
+    "test:requests": "jest --config=test/jest-config/jest.config-backend.js --testPathPattern='.*\\/generate-transactions\\.spec\\.ts'",
```

Comment:
(nitpick): can we have a config file for this test run please even if it just import/exports config backend for consistency